docs/source/conf.py

```diff
@@ -63,11 +63,11 @@
 
 autoclass_content = "both"
 
-# This dictionary should be of the form "bad type" -> "good type", and
-# replacements will be performed globally. Hyperlinks will be added in the
-# signature if "good type" is a valid (potentially user defined) python type
```

Comment:
It'd be good to keep this original comment with the explanation for the hyperlinks

Reply:
Ah, good catch, I didn't intentionally change this, just got confused when rebasing, will reverse this change.

Reply:
I think you've changed too much now. The addition of `libsemigroups::BMat8` -> `BMat8` was good, it was just the comment that should've been reverted. Sorry - should've made that more clear!